src/main/java/uk/gov/hmcts/reform/hmc/config/SecurityConfiguration.java

```diff
@@ -19,7 +20,7 @@ public class SecurityConfiguration {
         "/swagger-resources/**",
         "/swagger-ui/**",
         "/webjars/**",
-        "/v2/api-docs",
+        "/v3/api-docs",
```

Comment:
sorry for creating merge hell i just saw this, but you need 
```suggestion
        "/v3/api-docs/**",
```
or you'll get  GET http://hmc-hmi-inbound-adapter-aat.service.core-compute-aat.internal/v3/api-docs/swagger-config 401 (Unauthorized)
just fixed this in master

Reply:
Fixed in recent master merge [d48dbcb]